Connectors/Rhino/Speckle.Connectors.RhinoShared/Bindings/RhinoSendBinding.cs

```diff
@@ -316,34 +343,38 @@ private async Task RunExpirationChecks()
       }
     }
 
-    if (ChangedObjectIds.IsEmpty && ChangedObjectIdsInGroups.IsEmpty)
+    if (ChangedObjectIds.IsEmpty && ChangedObjectIdsInGroupsOrLayers.IsEmpty)
     {
       return;
     }
 
     // Actual model card invalidation
     string[] objectIdsList = ChangedObjectIds.Keys.ToArray();
-    var changedObjectIdsInGroups = ChangedObjectIdsInGroups.Keys.ToArray();
+    var changedObjectIdsInGroups = ChangedObjectIdsInGroupsOrLayers.Keys.ToArray();
```

Comment:
i would rename from `changedObjectIdsInGroups` to `changedObjectIdsInGroupsOrLayers` variable too to not confuse ourselves later